runtime/ms_rest_azure/spec/azure_service_client_spec.rb

```diff
@@ -64,13 +64,13 @@ module MsRestAzure
       azure_service_client.long_running_operation_retry_timeout = 0
 
       allow_any_instance_of(MsRestAzure::PollingState).to receive(:create_connection).and_return(nil)
-      allow(azure_service_client).to receive(:update_state_from_location_header) do |request, polling_state|
+      allow(azure_service_client).to receive(:update_state_without_custom_deserialization) do |request, polling_state|
```

Comment:
similar question here (as previous), the method here, does not differentiate aync_operation_header versus location_header, how are we validating the intent of the test now? and with the new logic are we still complying with the intent of the test?